### PR TITLE
Ndbex 63627 burial review page

### PR DESCRIPTION
**Pull request by Midge-dev**
## Summary

_(Summarize the changes that have been made to the platform)_
- Replace  the online Burials form with approved review page letting Veterans know the form isn't currently working and they to complete by mail.

_(What is the solution, why is this the solution)_
- The purpose of this page is to dynamically display any "In progress" form data to the Veteran so they can refer to it when filling out the paper form. This page is temporary while this online form is updated.

_(Which team do you work for, does your team own the maintenance of this component?)_
- Non-disability Benefits Experience

## Related issue(s)

[Burial | Build Review Page #63627](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63627)

## Testing done

- Manual testing for static elements
- Accessibility

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |    Screenshot from 2023-08-28 13-26-16    | Screenshot from 2023-08-28 13-24-49  |

## What areas of the site does it impact?
`/burials-and-memorials/application/530/introduction`
*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

- New keyboard-only E2E is still in development. Could use feedback on what should go in to this test regarding accessibility.

### CODE REVIEW

File: src/applications/burials/components/NoFormPage.jsx [hunk 1]
```diff
@@ -0,0 +1,486 @@
...
+];
+
+const generateData = (type, formData) => {
+  switch (type) {
+    case 'claimant-information':
+      return {
+        'Claimant’s first name': formData.claimantFullName.first
+          ? formData.claimantFullName.first
+          : '',
+        'Claimant’s middle name': formData.claimantFullName?.middle ?? 'None',
```
Comment (DLarson-Oddball) [inline src/applications/burials/components/NoFormPage.jsx:72]: The optional chaining guard on L72 isn't present on L69. Also, I love the use of nullish coalescing vs the ternary where the first path returns the same value compared in the conditional. Feels notably more succinct.

File: src/applications/burials/components/NoFormPage.jsx [hunk 2]
```diff
@@ -0,0 +1,486 @@
...
+              <br />
+              PO Box 5365
+              <br />
+              Janesville, WI 53547-5365
+              <br />
+            </p>
+            <article>
+              <va-on-this-page />
+              {renderFields.map(props => (
+                <CreateSummarySections
```
Comment (DLarson-Oddball) [inline src/applications/burials/components/NoFormPage.jsx:376]: What do you think about renaming this component `SummarySection`?

File: src/applications/burials/components/NoFormPage.jsx [hunk 3]
```diff
@@ -0,0 +1,486 @@
...
+            <hr className="vads-u-border-color--primary vads-u-margin-y--0 vads-u-border-bottom--2px" />
+            <p>
+              Call us at <va-link href="tel:800-827-1000" text="800-827-1000" />
+              . We’re here Monday through Friday, 8:00 a.m to 9:00 p.m ET. If
+              you have hearing loss, call TTY:{' '}
+              <va-link href="tel:711" text="711" />.
+            </p>
+          </div>
+        </>
+      ) : (
```
Comment (DLarson-Oddball) [inline src/applications/burials/components/NoFormPage.jsx:421]: This feels like a lot of logic to wrap in a ternary. Could we break this up into a couple functions returning the proper content instead?

File: src/applications/burials/reducers/itf.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
+// import set from 'platform/utilities/data/set';
```
Comment (DLarson-Oddball) [inline src/applications/burials/reducers/itf.js:1]: Do we still need this file?

**Review by DLarson-Oddball — CHANGES_REQUESTED** [review]
Pulled down and was able to review my answers to an in progress form

File: src/applications/burials/components/NoFormPage.jsx [hunk 5]
```diff
@@ -0,0 +1,486 @@
...
+            <va-link
+              download
+              filetype="PDF"
+              href="https://www.vba.va.gov/pubs/forms/VBA-21P-530EZ-ARE.pdf"
+              pages={8}
+              text="Download VA form 21P-530EZ"
+            />
+            <p>
+              We’ve captured your intent to file date of{' '}
+              <strong>XX/XX/XXXX</strong>. You have 12 months from that date to
```
[the remark this answers is not in the document]
Comment (Midge-dev) [reply src/applications/burials/components/NoFormPage.jsx:358]: Not at this time. API needs to give this to us

File: src/applications/burials/BurialsApp.jsx [hunk 6]
```diff
@@ -1,12 +1,30 @@
...
 import React from 'react';
+// import { Switch, Route, Link, useHistory, useLocation } from 'react-router-dom';
 
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
 import formConfig from './config/form';
+import { NoFormPage } from './components/NoFormPage'; // Make sure the path to NoFormPage is correct
 
 export default function BurialsEntry({ location, children }) {
-  return (
+  const showForm = false;
```
[the remark this answers is not in the document]
Comment (Midge-dev) [reply src/applications/burials/BurialsApp.jsx]: This was updated to use a flipper and has since been resolved. Thanks!

**Review by DLarson-Oddball — APPROVED** [review]

**Comment by Midge-dev** [comment]
This is ready for review. Thanks!

@department-of-veterans-affairs/vsa-debt-frontend
vsa-debt-frontend
@department-of-veterans-affairs/benefits-team-1-frontend
benefits-team-1-frontend
@department-of-veterans-affairs/platform-design-system-fe
platform-design-system-fe
@department-of-veterans-affairs/platform-va-product-forms
platform-va-product-forms
@department-of-veterans-affairs/va-platform-cop-frontend
VA Platform CoP Frontend

**Review by Andrew565 — APPROVED** [review]

**Review by Mottie — APPROVED** [review]

File: src/applications/burials/BurialsApp.jsx [hunk 7]
```diff
@@ -1,12 +1,48 @@
...
+import { NoFormPage } from './components/NoFormPage';
 
-export default function BurialsEntry({ location, children }) {
+function BurialsEntry({ location, children, isLoadingFeatures }) {
+  const { useToggleValue, TOGGLE_NAMES } = useFeatureToggle();
+  const burialFormEnabled = useToggleValue(TOGGLE_NAMES.burialFormEnabled);
+  const redirectToHowToPage =
+    burialFormEnabled === false && location.pathname !== '/introduction';
+  if (redirectToHowToPage === true) {
+    window.location.href = '/whatever/the/how-to/page/is';
```
[the remark this answers is not in the document]
Comment (Midge-dev) [reply src/applications/burials/BurialsApp.jsx]: @Mottie thanks for that! Fixed and pushed

**Pull request merged**